test/qweb3.js

```diff
@@ -511,5 +512,23 @@ describe('Qweb3', () => {
         assert.isDefined(res);
       });
     });
+
+    describe('backupWallet()', () => {
+      it('backup the wallet', async () => {
+        const res = await qweb3.backupWallet(path.join(__dirname, './data/backup.dat'));
+        assert.notTypeOf(res, 'Error');
+      });
+    });
+
+    describe('importWallet()', () => {
+      it('imports keys from a wallet dump file', async () => {
+        try {
+          await qweb3.importWallet(path.join(__dirname, './data/wallet.dat'))
+        }
+        catch (err) {
+          assert.isDefined(err)
```

Comment:
another thing you could also check in addition to `isDefined`, is `assertEquals` the error message text.